qiskit/quantum_info/synthesis/two_qubit_decompose.py

```diff
@@ -377,7 +377,7 @@ def decomp0(target):
         U0l = target.K1l.dot(target.K2l)
         U0r = target.K1r.dot(target.K2r)
 
-        return U0r, U0l
+        return np.around(U0r, 13), np.around(U0l, 13)
```

Comment:
it seems like there are `.dot` calls everywhere. I think would it better to round the result of the decomposition. Here is what I mean https://github.com/Qiskit/qiskit-terra/pull/4684

Reply:
The failure on windows with numpy 1.19.0 was very specific. It was only for `decomp0` (and only with `np.eye(4)` as the unitary being decomposed). There are no issues with the other methods. The rounding is unnecessary for other methods and will just slow them down.